streamer/src/nonblocking/connection_rate_limiter.rs

```diff
@@ -1,6 +1,15 @@
 use {
```

Comment:
doc: please a comment on the module level `//! This module introduces ... which allows to ...`

Reply:
Do we intend to kill Governor-based rate limiter? If not such comment will look awkward=) I think it is best left for another PR where we introduce this for use in TPU.